src/main/java/io/confluent/connect/elasticsearch/Validator.java

```diff
@@ -269,6 +270,16 @@ private void validateConnection(RestHighLevelClient client) {
     String exceptionMessage = "";
     try {
       successful = client.ping(RequestOptions.DEFAULT);
+    } catch (ElasticsearchStatusException e) {
+      switch(e.status()) {
+        case FORBIDDEN:
+          // ES is up, but user is not authorized to ping server
+          successful = true;
+          break;
+        default:
+          successful = false;
+          exceptionMessage = String.format("Error message: %s", e.getMessage());
```

Comment:
```suggestion
          throw e;
```

this will be recaught below and we can reuse the same logic

Reply:
IMHO it won't be recaught below because its a cascading exception block and not a nested try/catch

Reply:
It won't be recaught, +1 @kpatelatwork .